src/socket.cpp

```diff
@@ -365,6 +365,8 @@ void CSocket::OnDataReceived()
         use the signal/slot mechanism (i.e. we use messages for that).
     */
 
+    CHostAddress RecHostAddr;
+
```

Comment:
There's a comment down on line 401 I think should be against the whole block starting at 386, if I'm not mistaken.

Reply:
Yes, I think you're right; I'll move it.